ext/mixed/css/display.css

```diff
@@ -611,6 +611,102 @@ button.action-button {
     stroke-width: 5;
 }
 
+.term-glossary-image-container {
+    display: inline-block;
+    white-space: nowrap;
+    max-width: 100%;
+    position: relative;
+    vertical-align: top;
+    line-height: 0;
+    font-size: 0.07142857em; /* 14px => 1px */
+    overflow: hidden;
+}
+
+.term-glossary-image-link {
+    cursor: inherit;
+    color: inherit;
+}
+
+.term-glossary-image-link[href]:hover {
+    cursor: pointer;
+}
+
+.term-glossary-image-container-overlay {
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    font-size: 14em; /* 1px => 14px; */
+    line-height: 1.42857143; /* 14px => 20px */
+    display: table;
+    table-layout: fixed;
+    white-space: normal;
+}
+
+.term-glossary-item[data-has-image=true][data-image-load-state=load-error] .term-glossary-image-container-overlay:after {
+    content: "Image failed to load";
+    display: table-cell;
+    width: 100%;
+    height: 100%;
+    vertical-align: middle;
+    text-align: center;
+    padding: 0.25em;
+}
+
+.term-glossary-image {
+    display: inline-block;
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    vertical-align: top;
+    object-fit: contain;
+    border: none;
+    outline: none;
+}
+
+.term-glossary-image:not([src]) {
+    display: none;
+}
+
+.term-glossary-image[data-pixelated=true] {
+    image-rendering: auto;
+    image-rendering: -moz-crisp-edges;
+    image-rendering: -webkit-optimize-contrast;
+    image-rendering: pixelated;
+    image-rendering: crisp-edges;
+}
+
+.term-glossary-image-aspect-ratio-sizer {
+    content: "";
+    display: inline-block;
+    width: 0;
+    vertical-align: top;
+    font-size: 0;
+}
+
+.term-glossary-image-link-text:before {
+    content: "[";
+}
+
+.term-glossary-image-link-text:after {
+    content: "]";
+}
+
+:root[data-compact-glossaries=true] .term-glossary-image-container {
+    display: none;
+}
```

Comment:
Maybe they could pop out on mouse hover and show a thumbnail otherwise when compact glossaries is enabled

Reply:
So there are two ways I could go about doing this. I am trying to do it without adding any additional Javascript, because that runs into the issue of adding/removing event listeners, which I would like to put off until it becomes necessary. (It may become necessary for some UI design changes I would like to make, but not yet.)

1. Small, matching the width of the "[Image]" text.
   ![image](https://user-images.githubusercontent.com/11037431/79620575-4bbd1780-80de-11ea-886e-8101f6976b70.png)
2. Large, matching the width of the `<li>` element.
   ![image](https://user-images.githubusercontent.com/11037431/79620609-6abba980-80de-11ea-8b7d-b015d5e1cafa.png)

Let me know which is better.

Reply:
Assuming it's possible to use the `:hover` selector, I think it would be optimal if the element was visible when mouse is over the `[Image]` link. It could work similar to `termTags` in merge mode. If possible, I think the position could be in the center (`position: fixed;`) of the popup and the size limit could be something almost the width and height of the viewport. I think `pointer-events: none;` could help when mouse cursor is above the image but I didn't test.

Reply:
Right, I should have clarified that the images I posted are using mouse hover, and the mouse is hovered over the link in that case. So both of the two are possible using `:hover`.

Reply:
Ok. Unless you think that it's bad UX to make the hover image possibly overlap with the link, I think fixed position would be the most practical at least inside popups (but maybe not the search page). It would also be possible to display a small image inline but that could be undesirable for those who want it to be really compact. As long as it's possible to change the behavior with custom CSS we should have everyone covered, though.

Reply:
I currently went with the small image positioned relative to the [Image] link. IMO the large versions feel intrusive, and the small version fits more with the idea of being "compact". There are also positioning/sizing issues if trying to do the `position: fixed` version, primarily due to trying to deal with aspect ratio in CSS. So this is probably the most simple method until we get user feedback about it.

07e5e5c15b20077fd325b2546ae96c5a50d4c1bd

Reply:
Looks good :+1: My main issue was having to leave the entire page to even get an idea what the image is about. This fixes that part, but potentially makes the user do an additional step when the popup size would have been enough to understand the image contents completely. I didn't realize that there would be issues with aspect ratio and I agree we should wait for feedback before putting big effort into making it work.

I noticed that the small image will stay open when you click the link or the image and close the image's tab afterwards in Chrome. This could be a Chrome issue or a CSS issue. I didn't test in Firefox because the database upgrade would cause issues :smile:

Reply:
> I noticed that the small image will stay open when you click the link or the image and close the image's tab afterwards in Chrome. This could be a Chrome issue or a CSS issue. I didn't test in Firefox because the database upgrade would cause issues 😄

This is due to `:focus`, which I added so that you can tab to the link to see the preview.

Reply:
Ok, I think it's good to have it then.